cypress/integration/sidebar.js

```diff
@@ -0,0 +1,57 @@
+context('Testing Sidebar Functionality', () => {
```

Comment:
```suggestion
context('Sidebar', () => {
```

context can be just `Sidebar` or `Form Sidebar`... please check other context texts as well.